Modules/Core/Common/wrapping/test/itkPointSetTest.py

```diff
@@ -32,20 +33,23 @@ def test_SetPointsByCoordinates_from_NumPy_array(self) -> None:
         # Test the most common dimensions, and a few different numbers as number of points.
         for dimension in [2, 3]:
             for number_of_points in [1, 4, 9]:
+                point_set = itk.PointSet[PixelType, dimension].New()
+
                 # Create a NumPy array of random coordinates between -1.0 and 1.0
                 np_array = 2.0 * np.random.rand(number_of_points * dimension) - 1.0
 
-                point_set = itk.PointSet[PixelType, dimension].New()
+                # Set with flat array
+                point_set.SetPointsByCoordinates(np_array)
+
+                # Set with reshaped array
+                np_array = np.reshape(np_array, (number_of_points, dimension))
                 point_set.SetPointsByCoordinates(np_array)
 
-                points = point_set.GetPoints()
+                points = point_set.GetPointsByCoordinates()
 
-                self.assertEqual(points.Size(), number_of_points)
+                self.assertEqual(points.shape[0], number_of_points)
 
-                for i in range(number_of_points):
-                    point = points.GetElement(i)
-                    for j in range(dimension):
-                        self.assertAlmostEqual(point[j], np_array[i * dimension + j])
```

Comment:
I think this is a good test for the flat array case. I think it should not have been removed.

Reply:
We do retain testing for the flat array input. IT has the same output as the shaped array, and it is tested with the more detailed and efficient method.

Reply:
Thanks Matt, but in order to properly test for the flat array input, the asserts should be placed _directly_ after the `SetPointsByCoordinates(<flat array>)` call. Not just after the `SetPointsByCoordinates(<shaped array>)` call.